### PR TITLE
Fix internal links

### LINKED ISSUE #27: Fix internal links
Right now all the internal links have two problems:

- They all include the `.md` file extensions but should not.
- The all use `/` as the repo root but they should be relative.

E.g. `[link](../runtime/some-page)`

### PR DESCRIPTION
**Pull request by danforbes**
Closes #27

Thank you for making a PR to the knowledge base. Please make sure it follows these short guidelines:

- [x] Are the audience and objective of the document clear?
- [x] Is the writing:
  - **Clear**: All jargon defined, explained, or linked to. Best to avoid it.
  - **Precise**: No ambiguous meanings
  - **Concise**: Free of superfluous detail
- [x] Does it follow our [style guide](/CONTRIBUTING.md#documentation-style)?
- [ ] If this is a new page, does the PR include the appropriate infrastructure, e.g. adding the
  page to a sidebar?
- [ ] Build the page. Does it render properly? E.g. no funny lists or formatting.
- [x] Links should go to rustdocs or devhub articles, not source code.
- [x] If this PR addresses an issue in the queue, have you referenced it in the description?

### CODE REVIEW

File: current/runtime/events.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ decl_event!{
 }
 ```
 
-The `Event` enum needs to be declared in your runtime's [configuration trait](/current/runtime/traits.md).
+The `Event` enum needs to be declared in your runtime's [configuration trait](../runtime/traits).
```
Comment (joepetrowski) [inline current/runtime/events.md]:
```suggestion
The `Event` enum needs to be declared in your runtime's [configuration trait](traits).
```

File: current/runtime/frame.md [hunk 2]
```diff
@@ -287,7 +287,7 @@ a structure for making spending proposals from this pot.
 
 ### Learn More
 
-- Learn how to [develop custom Substrate pallets](/current/runtime/pallet.md).
+- Learn how to [develop custom Substrate pallets](../runtime/pallet).
```
Comment (joepetrowski) [inline current/runtime/frame.md]:
```suggestion
- Learn how to [develop custom Substrate pallets](pallet).
```

File: current/runtime/macros.md [hunk 3]
```diff
@@ -36,12 +36,12 @@ However, if you are interested in learning more details about what exactly the m
 
 ### Learn More
 
-* Learn how to use the [`decl_storage` macro](/current/runtime/storage.md).
-* Learn how to use the [`decl_event` macro](/current/runtime/events.md).
+* Learn how to use the [`decl_storage` macro](../runtime/storage).
+* Learn how to use the [`decl_event` macro](../runtime/events).
```
Comment (joepetrowski) [inline current/runtime/macros.md]:
```suggestion
* Learn how to use the [`decl_storage` macro](storage).
* Learn how to use the [`decl_event` macro](events).
```

File: current/runtime/macros.md [hunk 4]
```diff
@@ -36,12 +36,12 @@ However, if you are interested in learning more details about what exactly the m
...
 
 ### Examples
 
-View our most [simple Substrate runtime](/current/runtime/index.md) to see all these macros interact with one another.
+View our most [simple Substrate runtime](../runtime/index) to see all these macros interact with one another.
```
Comment (joepetrowski) [inline current/runtime/macros.md]:
```suggestion
View our most [simple Substrate runtime](index) to see all these macros interact with one another.
```

File: current/runtime/primitives.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ There are an additional set of primitives that are assumed about a runtime built
 
 ### Learn More
 
-- Learn about the [Substrate FRAME](/current/runtime/frame.md).
+- Learn about the [Substrate FRAME](../runtime/frame).
```
Comment (joepetrowski) [inline current/runtime/primitives.md]:
```suggestion
- Learn about the [Substrate FRAME](frame).
```

**Review by joepetrowski — APPROVED** [review]
Don't need the `../` when the link is in the same directory. (Only suggested changes for about half, you get the idea)

**Comment by danforbes** [comment]
Good catch, @joepetrowski, sorry about that...should be all set now :+1:

**Pull request merged**